.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Let us know about an unexpected error, a crash, or an incorrect behavior.
+labels: bug
+---
+
+<!--
+Hi there,
+
+Thank you for opening an issue! Please note that we try to keep the this issue tracker reserved for
+bug reports and feature requests related to the go-tfe API wrapper. If you know
+your issue relates to the Terraform Cloud/Enterprise platform itself, please contact
+tf-cloud@hashicorp.support. For general usage questions, please post to our community forum:
+https://discuss.hashicorp.com.
+-->
+
+#### go-tfe version
+<!---
+What version of go-tfe are you using?
+-->
+
```

Comment:
What do you think about adding a plaintext section here and for `Description`, `Testing plan`?
Like:
```plaintext
...
```

Reply:
I'm adding it for the go-tfe version and testing plan. We don't seem to have a consistent way or reason that we include plaintext fields in the templates.